src/model.cpp

```diff
@@ -60,11 +64,25 @@ std::string Model::serialise(libcellml::CELLML_FORMATS format) const
     std::string repr = "";
     if (format == CELLML_FORMAT_XML)
     {
-        repr += "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<model xmlns=\"http://www.cellml.org/cellml/1.2#\"></model>";
+        repr += "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<model xmlns=\"http://www.cellml.org/cellml/1.2#\"";
+        if (mPimpl->mName != initial_name)
+        {
```

Comment:
We want ```{``` at the end of the previous line (see https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Conditionals).